frontend/src/pages/AccountManagement/components/MyGroup.scss

```diff
@@ -0,0 +1,3 @@
+.my-group--group-name {
+    max-width: 250px;
```

Comment:
Each table width is set to USWDS 'maxw-tablet-lg' so to prevent squishing the action items (view, edit, delete) we set a max-width on the first column. We might ultimately need to set this on the Last Updated column.

Reply:
removed for now.